src/MgvOfferMaking.sol

```diff
@@ -326,7 +304,8 @@ contract MgvOfferMaking is MgvHasOffers {
         }
       }
 
-      // mapping (uint => MgvStructs.OfferPacked) _offers = offers[ofp.outbound_tkn][ofp.inbound_tkn];
+      Tick insertionTick = TickLib.fromLogPrice(insertionLogPrice, ofp.olKey.tickScale);
```

Comment:
It feels like we're converting back-and-forth between ticks and log prices. Maybe we should calculate both once and then drag them around in a struct?

Reply:
I forgot to adapt the insertion code to the fact that `offer.tick()` is now a derived value

FYI I think expanding our memory structs are costly --  e.g. with yesterday I was replacing `out,in,scale` with `key` in the events we emit, and I tried a) recompute `olKey.hash()` every time or b) add an `olKeyHash` field sor `SingleOrder` and `OfferPack`. The `market_order_8` gas test was 6k gas more expensive with b)!

Reply:
Understood 👍